### PR TITLE
feat: Documentation

### PR DESCRIPTION
**Pull request by ktquez**
Hi Kevin, how are you?

This PR is just a suggestion for the package to have documentation made with Vuepress.

If it is not interesting for you, no problem, you can close this PR.

If you are thinking of approving, I would like you to check it out and change anything you want.

Here's a [preview](https://epva.surge.sh/) of how it looks.

---

You can finish and publish by running the commands `npm run docs:build` and  `npm run docs:publish`

I set up a subdomain in the `vue-a11y.com` and will be available in [https://eslint-plugin.vue-a11y.com](https://eslint-plugin.vue-a11y.com) as soon as you publish the documentation.

### CODE REVIEW

**Comment by ktquez** [comment]
Sorry, I used npm, I didn't see that yarn was being used.

Can you upload the updated yarn.lock when testing the documentation locally?

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /.eslintcache
 /node_modules/
+docs/.vuepress/dist
```
Comment (TheJaredWilcurt) [inline .gitignore:3]:
```suggestion
docs/.vuepress/dist

```

Comment (ktquez) [reply .gitignore:3]: ?

Comment (TheJaredWilcurt) [reply .gitignore:3]: Plain text files must end in an empty return to prevent EOL issues in git.

Comment (ktquez) [reply .gitignore:3]: Perfect

**Comment by ktquez** [comment]
@TheJaredWilcurt 
Does it look good to you how will the documentation look?
Anything else you would like to have?

https://epva.surge.sh/

**Pull request closed without merging**